companion/manifest.json

```
@@ -3,7 +3,7 @@
 	"name": "rossvideo-rosstalk",
 	"shortname": "rosstalk",
 	"description": "rosstalk plugin for Companion",
-	"version": "0.0.0",
+	"version": "2.1.0",
```

This can stay as zero and it automagically populates the correct value!
```suggestion
	"version": "0.0.0",
```